docs/Contributing/API-for-contributors.md

```diff
@@ -961,7 +961,7 @@ Before you retrieve the live query results, you must create a live query campaig
 
 Note that live queries are automatically cancelled if this method is not called to start retrieving the results within 60 seconds of initiating the query.
 
-`/api/v1/fleet/results/websockets`
+`/api/latest/fleet/results/websockets`
```

Comment:
Based on the latest convos on Slack, looks like we want to stick to `v1` in the docs for now, so I'll change that back to `v1`. Quoting Noah:

> We’d like this “API for contributors” documentation page to also look like the way it did a couple weeks ago (using the v1 prefix).